packages/client/src/Feed.js

```diff
@@ -23,22 +23,22 @@ const Feed = () => {
   useEffect(() => {
     getAdminStatus(user.userId);
     populateFeed(isAdmin);
-  }, [isAdmin, posts]);
+  }, [posts]);
```

Comment:
You probably still want `isAdmin` here, is there a reason that `isAdmin` is unnecessarily updating?

Reply:
We changed the way the useEffect, it's dependencies and the getAdminStatus function were working in order to solve the issue of an infinitely re-rendering feed.